src/locales/pl/PAGES/EXPLORE.json

```diff
@@ -3,16 +3,16 @@
     "INTRO": "Przejrzyj witrynę, aby zrozumieć jej cel, funkcjonalność i sposób użytkowania. Przegląd witryny pomoże Ci ustalić, które strony internetowe poddać ocenie w kolejnych krokach. Określ technologie internetowe wykorzystane do stworzenia witryny i, jeśli chcesz, zanotuj uwagi na temat innych aspektów witryny. Jeśli nie wiesz, jakie technologie zostały użyte do stworzenia witryny, zapytaj o to osoby zajmujące się witryną oraz te, które ją zaprojektowały.",
     "RESOURCE_LINK_NAME": "Krok 2: Poznaj ocenianą witrynę",
     "HD_RELIEDUP_TECH": "Zastosowane technologie internetowe",
-    "INF_RELIEDUP_TECH": "Określ, jakie technologie internetowe [obsługujące dostępność zgodnie z WCAG 2](https://www.w3.org/Translations/WCAG21-pl/#dfn-obs-ugujaca-dostepnosc) zastosowano na witrynie. Aby uzyskać więcej informacji, zobacz [WCAG-EM Krok 2.d: Określ technologie, na których można polegać](https://testy.lepszyweb.pl/met-wcag-em-pl#step2d).",
+    "INF_RELIEDUP_TECH": "Określ, jakie technologie internetowe [obsługujące dostępność zgodnie z WCAG 2](https://www.w3.org/TR/WCAG21/#dfn-relied-upon) zastosowano na witrynie. Aby uzyskać więcej informacji, zobacz [WCAG-EM Krok 2.d: Określ technologie, na których można polegać](https://www.w3.org/TR/WCAG-EM/#step2d).",
```

Comment:
@zwiastunsw It is ok to link to the WCAG 2.1 Authorized Translation into Polish. [More info in the translation notes.](https://github.com/w3c/wai-wcag-em-report-tool/wiki/How-to-add-a-language#add-wcag-translations)